examples/storm-elasticsearch-examples/pom.xml

```diff
@@ -53,6 +54,11 @@
             <artifactId>storm-elasticsearch</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.elasticsearch</groupId>
+            <artifactId>elasticsearch</artifactId>
+            <version>${elasticsearch.test.version}</version>
```

Comment:
can you add the scope to provided. This will allow users to pick their own version of elasticsearch2 while packaging the topology